kqueen/engines/gce.py

```diff
@@ -158,18 +236,31 @@ def deprovision(self, **kwargs):
         result, error = super(GceEngine, self).deprovision(**kwargs)
         if result:
             return result, error
-
+        request = self.client.projects().zones().clusters().delete(projectId=self.project,
+                                                                   zone=self.zone,
+                                                                   clusterId=self.cluster_id)
         try:
-            self.client.projects().zones().clusters().delete(projectId=self.project, zone=self.zone, clusterId=self.cluster_id).execute()
+            request.execute()
             # TODO: check if provisioning response is healthy
         except Exception as e:
-            msg = 'Deleting cluster {} failed with following reason: {}'.format(self.cluster_id, repr(e))
+            msg = 'Deleting cluster {} failed with following reason: {}'.format(self.cluster_id,
+                                                                                repr(e))
             logger.exception(msg)
             return False, msg
 
         return True, None
 
     def resize(self, node_count, **kwargs):
+
+        if int(node_count) < 2 and \
+           self.cluster_config['cluster']['networkPolicy']['enabled'] is True:
```

Comment:
Seems like incorrect indentation

Reply:
everything is ok, flake passed, travis passed)